cmd/cluster/clusterCreate.go

```diff
@@ -123,7 +123,7 @@ func NewCmdClusterCreate() *cobra.Command {
 				l.Log().Fatalf("error processing/sanitizing simple config: %v", err)
 			}
 
-			clusterConfig, err := config.TransformSimpleToClusterConfig(cmd.Context(), runtimes.SelectedRuntime, simpleCfg)
+			clusterConfig, err := config.TransformSimpleToClusterConfig(cmd.Context(), runtimes.SelectedRuntime, simpleCfg, configFile)
```

Comment:
@iwilltry42 Do you know a better way to get the `configFile` without adding it here as an extra arg? :thinking:

Reply:
I have no problem with adding it as an extra parameter :)